lib/vsc/filesystem/oceanstor.py

```diff
@@ -310,6 +309,8 @@ def __init__(self, url, account, username, password):
         self.oceanstor_nfsservers = dict()
 
         self.vsc = VSC()
+        self.vscstorage = VscStorage()
+        self.institutes = [institute for institute in self.vscstorage if not institute.startswith('VSC')]
```

Comment:
I don't understand where this comes from? We have the config file set up for `BRUSSEL` it should just work no?

Reply:
Technically, the configuration can have settings from multiple sites. See https://github.com/vub-hpc/vsc-administration/blob/master/test/filesystem_info.conf

`VscStorage` returns a dict with keys per site (*e.g.* `brussel`, `gent`) plus some top level keys for default filesystems (*e.g.* `VSC_HOME`). Since we do not have the site information, this lists all sites available in the config to be able to loop through all listed filesystems.

Alternatively, we could just look into the default filesystems (top level `VSC*` keys). This will certainly work for us, not sure if it will in all possible cases.

Reply:
The top level key thing is there for backward compatibility (aka, not break Gent stuff). I prefer that you hard-code `BRUSSEL` here so it's at least clear that this does.

Reply:
And actually we do know, it's in the `VSC()` class from the `/etc/vsc_config.cfg` file: there is both `host_institute` and `site_name`. No clue any more what's the difference.

Reply:
We can take the host institute from the environment variable `VSC_INSTITUTE_LOCAL` and avoid either hardcoding `brussel` or having to read yet another conf file. Fixed in https://github.com/vub-hpc/vsc-filesystem-oceanstor/pull/21/commits/818d263c97bb7eeba56f52d39de2a9ea98537f9b

Reply:
no, that's a variable that is only guaranteed for VSC users. Not for our system users.

The ocnfig file is there and I think `VSC()` already reads it by default?

Reply:
you are right, fixed in https://github.com/vub-hpc/vsc-filesystem-oceanstor/pull/21/commits/624b4adcebeca03080ca8ca2afa9dc3fd5b7d4ba